tools/ClassGen/NodeBuilder.h

```diff
@@ -46,6 +46,7 @@ class NodeBuilder {
   std::vector<std::pair<std::string, std::string>> nodeOutputs_;
   /// A list of node members. Format: (type, name).
   std::vector<std::pair<MemberTypeInfo, std::string>> members_;
+  std::vector<std::pair<MemberTypeInfo, std::string>> variableOutputMembers_;
```

Comment:
Please add  a Doxygen comment for this member.

Reply:
Done.